docs/release-notes/highlights-1.4.0.asciidoc

```diff
@@ -0,0 +1,14 @@
+[[release-highlights-1.4.0]]
+== 1.4.0 release highlights
+
+[float]
+[id="{p}-140-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 1.4.0 of {n}. See <<release-notes-1.4.0>> for the full list of changes.
+
+[float]
+[id="{p}-140-agent-support"]
+==== Support for Elastic Agent
+
+link:https://www.elastic.co/guide/en/fleet/current/fleet-overview.html[Elastic Agent] is a single, unified way to add monitoring for logs, metrics, and other types of data to each Kubernetes node. With a single agent, it's easier and faster to deploy monitoring across your infrastructure; also there is no need to deploy multiple instances of Beats for different kinds of data you want to monitor. ECK 1.4.0 introduces experimental support for Elastic Agent in link:https://www.elastic.co/guide/en/fleet/current/run-elastic-agent-standalone.html[standalone mode] as a technology preview.
```

Comment:
I know the description comes from the product page. It just feels like a weird way of explaining things. Feel free to ignore my suggestion though.
```suggestion
link:https://www.elastic.co/guide/en/fleet/current/fleet-overview.html[Elastic Agent] provides a unified way to monitor logs, metrics, and other types of data from your Kubernetes infrastructure quickly and easily.  A single agent deployment can be used to replace multiple Beats deployments that were previously necessary to collect the different types of data you want to monitor. ECK 1.4.0 introduces experimental support for Elastic Agent in link:https://www.elastic.co/guide/en/fleet/current/run-elastic-agent-standalone.html[standalone mode] as a technology preview.
```